### ISSUE TITLE
The portal effect appears suddenly 

### ISSUE THREAD

**Issue by CarlosMiguelCuevas** [Nearsoft/office-portal#12]
Some times the portal effect appears from nowhere. @ivanebernal  suggested that it might be caused by the vending machine's compressor. We added a surge suppressor but it didn't work . @ivanebernal  thinks it is caused by an electromagnetic field caused by the vending machine that interferes with the raspberry's radio sensor.

**Comment by CarlosMiguelCuevas** [comment]
I think we can solve this issue and the issue #13 by removing the raspberry and instead use the wii-mote (the wii control) and also connect the LEDs directly to the power.

What do you think team? @paulina-pacheco , @ivanebernal , @hkmoca , @ar-vargas , @cesardariogarcia 
I only mentioned the HMO team because the Chihuahua team is already using the wii-mote.

**Comment by CarlosMiguelCuevas** [comment]
@echavezNS this is what I was talking about,  we can "kill two birds with one shot" if we use the wii-mote

**Comment by hkmoca** [comment]
yep, the problem is that if you use the wii mote to control the portal, the control is always ON, so the batteries ran out of power very quickly

**Comment by CarlosMiguelCuevas** [comment]
We checked the log when the portal turned white. this is what we found: 

- DOMException: Error processing ICE candidate

-undefined:1 Uncaught (in promise) DOMException: Failed to set remote offer sdp: Called in wrong state: STATE_SENTOFFER
